.github/workflows/ci.yml

```diff
@@ -0,0 +1,107 @@
+name: CI
+on:
+  push:
+    branches:
+      - '**'
+    tags-ignore:
+      - v*
+
+jobs:
+  commitlint:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Commitlint
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        uses: wagoid/commitlint-github-action@v2
+
+  word-blacklist:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Word Blacklist
+        env:
+          BLACKLIST: ${{ secrets.BLACKLIST }}
+        run: ./scripts/ci/word-blacklist.sh
+
+  test:
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 10.x
```

Comment:
Why node 10?

Reply:
Previously we had node 8 and 10. But 8 is not supported by our current set of dependencies anymore. Because of this I already had to disable the engine check in 2c46feb of #576. With #611 we also won't be able to use 8 anymore because prettier needs at least 10.

We could think about adding newer versions using a matrix.

PS: Removing node 8 from the pipelines might technically be considered breaking, since we can't officially support it anymore. Should we officially make this a 3.0.0 release because of this? I'm hesitant to do this.

Reply:
Sorry, I have not made myself clear.I am more interested in Node 12, which is supported by Lambda and is the default for aws-simple. Therefore we should test Node 12 and Node 10 as a matrix test.

Reply:
Got you. 👌 Note: I had to update jest, babel and tslint to be fully compatible with node 12.